tests/components/squeezebox/test_config_flow.py

```diff
@@ -0,0 +1,287 @@
+"""Test the Logitech Squeezebox config flow."""
+from asynctest import patch
+from pysqueezebox import Server
+
+from homeassistant import config_entries
+from homeassistant.components.squeezebox.const import DOMAIN
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    HTTP_UNAUTHORIZED,
+)
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+)
+
+HOST = "1.1.1.1"
+HOST2 = "2.2.2.2"
+PORT = 9000
+UUID = "test-uuid"
+UNKNOWN_ERROR = "1234"
+
+
+class MockEntry:
```

Comment:
There is an existing fixture for these
`from tests.common import MockConfigEntry`

Reply:
Nice. I switched it to this.